sandbox/modules/identity/build.gradle

```diff
@@ -32,10 +32,11 @@ dependencies {
  * We have to disable setting the number of available processors as tests in the same JVM randomize processors and will step on each
  * other if we allow them to set the number of available processors as it's set-once in Netty.
  */
-test {
-  systemProperty 'opensearch.set.netty.runtime.available.processors', 'false'
-}
+  test {
+    systemProperty 'opensearch.set.netty.runtime.available.processors', 'false'
+  }
 
-internalClusterTest {
-  systemProperty 'opensearch.set.netty.runtime.available.processors', 'false'
+  internalClusterTest {
+    systemProperty 'opensearch.set.netty.runtime.available.processors', 'false'
+  }
```

Comment:
Looks like an indentation issue caused due to missing parenthesis above (it has been misplaced at the line below)